services/galley/test/integration/API.hs

```diff
@@ -857,6 +863,31 @@ leaveConnectConversation = do
   let c = fromMaybe (error "invalid connect conversation") (cnvId <$> responseJsonUnsafe bdy)
   deleteMember alice alice c !!! const 403 === statusCode
 
+-- This test adds a non existent remote user and expects a 200. This is of
```

Comment:
I would rephrase as "adds a non-existent remote user *to a conversation*". I was confused at first, thinking this was creating a remote user.